src/test/java/school/cesar/eta/unit/LeapYearCalculatorTest.java

```diff
@@ -0,0 +1,141 @@
+package school.cesar.eta.unit;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.Year;
+
+
+public class LeapYearCalculatorTest {
+
+
+
+
+    @Test
+    public void isLeapYear_currentYearIsLeapYear() {
+        LeapYearCalculator calculator = new LeapYearCalculator();
+        Assertions.assertTrue(calculator.isCurrentYearLeapYear());
```

Comment:
Essa condição está utilizando a data do sistema e enquanto for 2020 o teste passará, mas na virada do ano começará a falhar.

Para evitar essa intermitências é necessário controlar o retorno da função [`getCurrentYear()`](https://github.com/cesar-school/unit-testing-project/blob/d8b1f46b3efe6833ad4b0cec877885168676b4dc/src/main/java/school/cesar/eta/unit/LeapYearCalculator.java#L21)

```java
LeapYearCalculator calculator = new LeapYearCalculator() {
    @override
    public int getCurrentYear() {
        return 2020;
    }
};
```